### ISSUE TITLE
Enhancements to the catalog tool

### ISSUE THREAD

**Issue by cbourget** [infra-geo-ouverte/igo2-lib#220]
* **Igo Version:**
0.24.0

These features are to be added to the catalog tool:

- [ ] Keep track of the layers already added to the map and identify them in the catalog's layers list.
- [ ] By clicking on a button, add all the layers of a group to the map at once.
- [ ] Optional: Don't add a layer right away with a single click on it. Instead, add a button to do the "add" and show some info about the layer on a single click.

I think this task would be easier done if the catalog tool used DataStore (see #219) to track the catalogs and their layers. This could be an opportunity to refactor the catalog tool.

This is an issue to track various subtasks.

**Comment by cbourget** [comment]
See my comment in #219.

TLDR:  It's done in an igo2 fork using the new entity module (still in dev mode).

Layers and groups can be added/removed by clicking on a button.
Layers that are already added are identified as such.

Showing layer info on select could be done quite easily if the need arises.

**Comment by cbourget** [comment]
Right now, they don't. But it would be pretty easy to do.
